src/pilet-build.test.ts

```diff
@@ -262,7 +262,7 @@ runTests('pilet-build', ({ test, setup }) => {
     await ctx.assertFiles({
       'dist/index.js'(content: string) {
         expect(content).not.toBe('');
-        expect(content).not.toContain('var moduleUrl');
+        expect(content).not.toContain('foobar1234');
```

Comment:
Yeah I would also expect it not to contain the string if the string is not even added to the source.